webpack.config.cjs

```diff
@@ -4,6 +4,78 @@ const { TsconfigPathsPlugin } = require( 'tsconfig-paths-webpack-plugin' );
 const FileManagerPlugin = require( 'filemanager-webpack-plugin' );
 const webpack = require( 'webpack' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const fs = require( 'fs' );
+
+// @TODO: Sample paths, need to update
+const SCHEMA_SRC_DIR = './schema';
+const SCHEMA_OUTPUT_NAME = 'schema.json';
+const SCHEMA_OUTPUT_PATHS = [ './dist/config', './public/config' ];
```

Comment:
These directories are currently created, but their not excluded from git. Could we move them under `build/`?

Reply:
I have made the change to move it to the hopefully right places where we would need it.